src/injected/analyzers/rule-analyzer.ts

```diff
@@ -71,7 +71,7 @@ export class RuleAnalyzer extends BaseAnalyzer {
     }
 
     protected onResolve = (analyzerResult: AxeAnalyzerResult): void => {
-        this.sendScanCompleteResolveMessage(analyzerResult, this.config);
+        this.sendScanCompleteResolveMessage(analyzerResult, this.config); // doesn't change analyzerResult to be what we want so... notifications etc. are with wrong info for NR
```

Comment:
Is this comment just a note-to-self that will be deleted? Otherwise, I think it could use a little more _why_. Also, 'NR' will not age well in a comment; You may not remember what it means six months from now. :)

Reply:
Yep, was a note to self.  Removed :)